### ISSUE TITLE
how to use my trained pytorch model into android project?

### ISSUE THREAD

**Issue by billhyde** [deepcam-cn/Face-Anti-spoofing.pytorch#9]
hi,thanks for sharing~ if I trained my own model ,how to convert it into android project and how to do inference in android? thanks~

**Comment by liurs1990** [comment]
you can save ONNX model and then transform to Caffe model.

**Comment by billhyde** [comment]
Hi,thank you for reply. Is your Android example use caffe model? If you did not use the caffe framework, would you tell me what Deep learning framework you use?
 I tested the android example ,it's pretty good, but i can not find the android infercence C++ source code. I want to use my own train pytorch model in android example like you,please help me,thanks .

**Comment by liurs1990** [comment]
No, I don't use Caffe on Android. NCNN is a good choice for Android.

**Comment by vwillagent** [comment]
we have trained a pytorch model and then convert to onnx model, the onnx model can be loaded and runned by python and get same predict score as pytorch  model. 
and then we convert onnx to ncnn using ncnn tool 'onnx2ncnn', we can load and run the ncnn model but predict is error, can you share you method, thanks

**Comment by PhenomenalOnee** [comment]
Can someone please provide your pretrained model with inference code in python

**Comment by luan1412167** [comment]
> we have trained a pytorch model and then convert to onnx model, the onnx model can be loaded and runned by python and get same predict score as pytorch model.
> and then we convert onnx to ncnn using ncnn tool 'onnx2ncnn', we can load and run the ncnn model but predict is error, can you share you method, thanks

Can you share how accurate you get with the model?
